### ISSUE TITLE
Group parameters are missing

### ISSUE THREAD

**Issue by Froxz** [glhd/gretel#7]
**Describe the bug**
By wrapping resource into a Route Group with {parameter}, throws an error that parameter doesn't exists,

I've tried to pass the parameter as 3rd callback, but the same result.

**What version does this affect?**
 - Laravel Version: [9.10.0]
 - Package Version: [1.5.0]

**To Reproduce**
```php
Route::resource('movies', 'MovieController')->except([
    'show'
])->breadcrumbs(function($breadcrumbs) {
    $breadcrumbs
        ->index('Movies')
        ->create('Create')
        ->edit('Edit', '.index');
});

Route::prefix('/movies/{movie}')->group(function () {

    Route::resource('actors', 'ActorController')->except([
        'index', 'show'
    ])->breadcrumbs(function($breadcrumbs) {
        $breadcrumbs
            ->create('Create', 'movies.index', fn($movie) => $movie)
            ->edit('Edit', 'movies.index', fn($movie) => $movie);
    });
    
});
```

**Expected behavior**
Generate breadcrumbs

**Additional context**
Throws an error:

Missing required parameter for [Route: actors.create] [URI: movies/{movie}/actors/create] [Missing parameter: movie].

**Comment by inxilpro** [comment]
Hm, I've dug into this a bit and can recreate the bug but it looks like it might be a bit tricky to sort out. We have to do some tricky stuff to have access to the route parameters at the right time, and it looks like some of that approach may need to be rethought to make nested groups work in this case.

**Comment by inxilpro** [comment]
This should be fixed in 1.6.0

**Comment by Froxz** [comment]
Hi @inxilpro apologies I didn't had time to check this, will do today and revert thank you.
